toolkit/offchain/src/reserve/release.rs

```diff
@@ -459,12 +460,12 @@ mod tests {
 			.get(&token_name().to_csl().unwrap())
 			.expect("Should transfer reserve token to illiquid supply");
 
-		assert_eq!(illiquid_supply_output, 10u64.into())
+		assert_eq!(illiquid_supply_output, 5u64.into())
 	}
 
 	#[test]
 	fn should_leave_unreleased_tokens_at_reserve_validator() {
-		let illiquid_supply_output = (reserve_release_test_tx().body().outputs().into_iter())
+		let validator_output = (reserve_release_test_tx().body().outputs().into_iter())
```

Comment:
Yeah, imagine I was wondering why after effectively decreasing the transfer amount from 10 to 5, I've seen the illiquid supply output to grow.